### PR TITLE
Query/availabletime

### PR DESCRIPTION
**Pull request by tasneembhiri**
### Content
 - add router for available time
 - create a query to get available time for a specified date 
-  create controller file  for available time 
- update customer_info table 
- create new table named reservation_time
-  add fixed data for reservation_time
- update post customer information query  
---

### Testing
on terminal use the following commands:
- pull origin Query/avaialbletime
- npm run server
- go to website https://reqbin.com/
- **After you fill all the information as shown on the picture below**

image

- **After sending the request you should receive a response** 

image

### CODE REVIEW

File: server/controllers/postCustomer.js [hunk 1]
```diff
@@ -2,9 +2,9 @@ const postCustomer = require('../database/queries/postCustomer');
...
 module.exports = (req, res) => {
   const data = req.body;
-  const { fullName, phone, email, zipCode, reserveDate } = data;
+  const { fullName, phone, email, zipCode, reservationDate, timeId } = data;
   // check if the all text fields is filled.
-  if (!fullName || !phone || !email || !zipCode || !reserveDate) {
+  if (!fullName || !phone || !email || !zipCode || !reservationDate || !timeId) {
     // if one of text fields is empty, show this message
     return res.status(400).json({
       message: 'There is one item or more missing. Please filled it!',
```
Comment (Jood80) [inline server/controllers/postCustomer.js]: This error message can be more descriptive, for instance you may say:
`Please make sure that you filled each field properly` 
However I believe this message still not that good so maybe we can change it later.

**Review by sara219 — APPROVED** [review]
:+1:

File: server/controllers/postCustomer.js (hunk 1 above)
Comment (Rahaf-96) [reply server/controllers/postCustomer.js]: descriptive :stuck_out_tongue_winking_eye:

**Review by Rahaf-96 — APPROVED** [review]
GREAT JOB! :star_struck: :star_struck:

**Review by Jood80 — APPROVED** [review]

**Pull request merged**